doc/reference/network_ovn.md

```diff
@@ -15,15 +15,8 @@ Set up OVN </howto/network_ovn_setup>
 Create routing relationships </howto/network_ovn_routing>
 ```
 
-Network forwards:
```

Comment:
How do we advertise in the network section that a network supports ACLs, forwards and peers?

Reply:
> How do we advertise in the network section that a network supports ACLs, forwards and peers?

We present those as general features (because we want people to use mainly bridge and OVN) and state clearly at the top of each feature page for which networks the feature is available.
I also have a note to add a clear "this is not supported" statement on the landing page for external networks.

Reply:
I haven't reworked the sections for the different network types yet. I'll make a note to add this.

Reply:
Thanks